### PR TITLE
Fix issue where node process hangs after successful dank meme generation.

### PR DESCRIPTION
**Pull request by Jaace**
# Description
Exiting the node process after successful memeing, with messaging. Fix some indentation issues.

## Steps to Test
Run the three types of meme generation, ensure that upon completion the command line is back to a standard prompt (not hanging).

Throw up a message to the terminal upon successful exit, similar to the exit messages on error or quitting.

- `node index.js`
- `node reddit.js`
- `node roulette.js`

## Checklist
- [x] I have tested these code changes and included the relevant urls and screenshots.
- [ ] I have updated our build tasks accordingly (if applicable).
- [ ] I have updated our documentation accordingly (if applicable).
- [ ] I have linted the code that I've included in this PR (if applicable).
- [x] My code follows the code style of this project.
- [x] My code is ready for review.

### CODE REVIEW

**Comment by Jaace** [comment]
This is ready for review :)

File: reddit.js [hunk 1]
```diff
@@ -9,12 +9,14 @@ getList();
...
-		}
+      // Basically doesn't display the image only if the filter is turned on AND the post is nsfw
+      if (filterNSFW == false && body.data.children[i].data.over_18 == false) {
+        opn(body.data.children[i].data.preview.images[0].source.url);
+      }
     }
   }
+  console.log('Pulling up some dank reddit memes...');
+  process.exit();
 }
```
Comment (cjwilburn) [inline reddit.js:20]: Thanks for cleaning this up!

File: roulette.js [hunk 2]
```diff
@@ -44,6 +45,8 @@ generateRandomlyCaptionedMeme = (caption1, caption2) => {
 			return r.data.url
 		})
 		.then(url => {opn(url)})
+		.then(() => process.exit())
+		.catch(console.error)
 	})
```
Comment (cjwilburn) [inline roulette.js:50]: looks good @Jaace

**Review by cjwilburn — APPROVED** [review]
LGTM!

**Pull request merged**